src/bp-notifications/bp-notifications-functions.php

```diff
@@ -482,7 +482,18 @@ function bp_notifications_delete_notifications_on_user_delete( $user_id ) {
 		'component_name'    => false,
 	) );
 }
-add_action( 'wpmu_delete_user', 'bp_notifications_delete_notifications_on_user_delete' );
+
+/**
+ * Callback: Delete a user's notifications when the user is deleted.
+ *
+ * @since 12.0.0
+ *
+ * @param int $user_id ID of the user who is about to be deleted.
+ */
+function bp_notifications_delete_notifications_on_user_delete_callback( $user_id ) {
```

Comment:
We can also name it `bp_notifications_delete_notifications_on_user_delete_action`. I see other functions in other components with similar nomemclature.

Reply:
Awesome! Thanks a lot for your work on this. I like your solution 💪. I'm fine with both function names.